bin/check_samplesheet.py

```diff
@@ -158,9 +158,6 @@ def sniff_format(handle):
     peek = read_head(handle)
     handle.seek(0)
     sniffer = csv.Sniffer()
-    if not sniffer.has_header(peek):
```

Comment:
hmm why?

Reply:
I have literally no idea. the first time it happened i tried rewriting in vim, opening and re-saving in excel, reading in and writing out in R, re-writing from scratch in excel...so I don't know what it failed.

Reply:
This just in! @elizabethmcd got this error on an nf-core MAG sample sheet so i feel like this is a script problem and not a me problem.

Reply:
yes basically any nf-core MAG samplesheet I try without this commented out it says it doesn't have a header, even though it does, and when I comment this out things go fine

Reply:
this is what the file looks like with it's special characters...is it missing an `\r` at the end of rows or something? does anyone spot anything?
```
bb =open(filename, "rb")
cc = bb.read()
cc
```
```
b'sample,fastq_1,fastq_2\nSRR8528939,https://github.com/Arcadia-Science/test-datasets/raw/main/seqqc/SRR8528939_10k.fq.gz,\nSRR11550236,https://github.com/Arcadia-Science/test-datasets/raw/main/seqqc/SRR11550236_1_1k.fq.gz,\nSRR3927310,https://github.com/Arcadia-Science/test-datasets/raw/main/seqqc/SRR3927310_1_10k.fq.gz,https://github.com/Arcadia-Science/test-datasets/raw/main/seqqc/SRR3927310_2_10k.fq.gz\n'
```

Reply:
Looks the same to me as one that doesn't produce this error
```
b'sample,fastq_1,fastq_2\ncomm_1,https://github.com/Arcadia-Science/test-datasets/raw/main/cheese-illumina-metagenomes/comm_1_subsampled_1.fq.gz,https://github.com/Arcadia-Science/test-datasets/raw/main/cheese-illumina-metagenomes/comm_1_subsampled_2.fq.gz\nvir_1,https://github.com/Arcadia-Science/test-datasets/raw/main/cheese-illumina-metagenomes/vir_1_subsampled_1.fq.gz,https://github.com/Arcadia-Science/test-datasets/raw/main/cheese-illumina-metagenomes/vir_1_subsampled_2.fq.gz\n'
```

Reply:
(im going to merge this for now)